Disable the analyze ATR perl-script and as we don't want to pull in perl
as a dependency.

--- a/pcsc_scan.c
+++ b/pcsc_scan.c
@@ -294,7 +294,7 @@ static void initialize_options(options_t
 #if defined(WIN32) || defined(__APPLE__)
 	options->analyse_atr = False;
 #else
-	options->analyse_atr = True;
+	options->analyse_atr = False;
 #endif
 	options->stress_card = False;
 	options->maxtime = 0;
